doc/template.html

```diff
@@ -45,5 +45,6 @@
   <script src="assets/sh_main.js"></script>
   <script src="assets/sh_javascript.min.js"></script>
   <script>highlight(undefined, undefined, 'pre');</script>
+  __TRACKING__
```

Comment:
can we perhaps comment this? `<!-- __TRACKING__ -->` -- that way we don't have to process any files if tracking is disabled.